zeeguu_api/api/teacher_dashboard.py

```diff
@@ -432,3 +436,81 @@ def update_cohort(cohort_id):
     except sqlalchemy.exc.IntegrityError:
         flask.abort(400)
         return 'IntegrityError'
+
+
+@api.route("/upload_files/<cohort_id>", methods=["POST"])
+@with_session
+def upload_file(cohort_id):
+    '''
+        uploads a file for a cohort with input from a POST request
+    '''
+    if (not has_permission_for_cohort(cohort_id)):
+        flask.abort(401)
+    try:
+        for file in json.loads(request.data):
+            url = Url('teacher-dash-file/{}'.format(datetime.now()))
+            title = file['title']
+            authors = file['authors']
+            content = file['content']
+            summary = file['summary']
+            published_time = datetime.now()
+            language_code = file['language_code']
+            language = Language.find(language_code)
+
+            new_article = Article(
+                url,
+                title,
+                authors,
+                content,
+                summary,
+                published_time,
+                None,  # rss feed
+                language
+            )
+            
+            db.session.add(new_article)
+            db.session.flush()
+            db.session.refresh(new_article)
+            
+            cohort = Cohort.find(cohort_id)
+            new_cohort_article_map = CohortArticleMap(cohort, new_article)
+
+            db.session.add(new_cohort_article_map)
+        db.session.commit()
+        return 'OK'
+    except ValueError:
+        flask.abort(400)
+        return 'ValueError'
+
+
+@api.route("/cohort_files/<cohort_id>", methods=["GET"])
+@with_session
+def cohort_files(cohort_id):
+    '''
+        Gets the files associated with a cohort
+    '''
+    cohort = Cohort.find(cohort_id) 
+    articles = CohortArticleMap.get_articles_for_cohort(cohort)
+    return json.dumps(articles)
+
+@api.route("/delete_article/<cohort_id>/<article_id>", methods=["POST"])
+@with_session
+def delete_article(cohort_id, article_id):
+		'''
```

Comment:
in the GH diff UI there's a lot of indentation in this method.
can you check why? 

<img width="1045" alt="Screen Shot 2019-04-02 at 2 47 20 PM" src="https://user-images.githubusercontent.com/464519/55403504-471ad500-5556-11e9-86b5-610954bcb446.png">

Reply:
I tried to fix this, but in the code it looks normal. Let me know if its still happening